evergreen-wellness/evergreen-wellness.php

```diff
@@ -83,3 +83,51 @@ function set_default_admin_color($user_id) {
 	add_action('user_register', 'set_default_admin_color');
 }
 
+/**
+ * Description: Production ID Admin sort and search
+ * Purpose: Add custom sort column for Production ID to POST and VIDEO content types
+ * Notes: Funtion relies on ACF plugin to work properly, field name must be "production_id"
+ * @author  etwilbeck
+ * @since  1.0.0
+ */
+add_filter('manage_posts_columns', 'egw_columns_head');
```

Comment:
right, but John didn't want to limit it to just video posts - so now it will show up on any actual post type, both posts, featured articles, and videos. And we can probably filter out any new ones we add eventually if needed.

But good catch!